addons/website_sale_delivery/models/sale_order.py

```diff
@@ -72,3 +73,14 @@ def _cart_update(self, *args, **kwargs):
         """ Override to update carrier quotation if quantity changed """
         self._remove_delivery_line()
         return super()._cart_update(*args, **kwargs)
+
+    def get_access_point_address(self):
+        if self.access_point_address:
+            return json.loads(self.access_point_address)
+        return False
+
+    def write(self, values):
+        access_point = values.get('access_point_address')
+        if access_point:
+            values['access_point_address'] = access_point
+        return super(SaleOrder, self).write(values)
```

Comment:
I have a question about that actually.

Shouldn't it be used instead of `order.write({'access_point_address': access_point})` in the controller?

I removed the method but now I have errors about concurrent updates of the sale order